Add documentation for jobs without resource requests (#34)
beaker/docs: docs/start/interactive.md

### Reserving GPUs

> By default, sessions aren't assigned a GPU.
> They are also not guaranteed any memory and may be killed if there is contention for memory on the machine.
> If you need GPUs, use the `--gpus <count>` flag when creating a session e.g.:

Reviewer: Sessions can claim limits the same way batch jobs do.

```suggestion
By default, sessions do not request any resources and won't be assigned a GPU.
If you need GPUs, use the `--gpus <count>` flag when creating a session e.g.:
```
